.github/copilot-instructions.md

```diff
@@ -1,9 +1,20 @@
-# General Principles
-
-1. **Follow the project style**: All changes should adhere to the existing code and documentation style.
-2. **Use Material Design colors**: All icons should use colors from the [Material Design color palette](/material-colors.yml).
-3. **Adhere to project architecture**: All changes should comply with the project architecture described in [architecture.md](/src/architecture.md).
-4. **Pixel-Perfect Icons**: Ensure that all icons are designed to be sharp and clear at 16x16 pixels, following the guidelines in [Designing Pixel Perfect Icons](/CONTRIBUTING.md#pixel-perfect-icons).
-5. **Write clean, modular, and well-documented code.**: All changes should be well-documented and easy to understand.
-6. **Test your changes**: All changes should be tested and should not break existing functionality.
-7. **Keep it simple**: All changes should be as simple as possible and should not introduce unnecessary complexity.
+To contribute effectively to this project, follow these guidelines:
```

Comment:
I would like to see a headline here? Or is it intentional that we start with the text right away?

Reply:
> I would like to see a headline here. Or is it intentional that we start with the text right away?

I can add the heading "Copilot instructions"

Reply:
yeah that would be nice :)

Reply:
Done